lib/constants.ts

```diff
@@ -16,3 +16,5 @@ export const TWITTER_URL = 'https://twitter.com/backed_xyz';
 export const GITHUB_URL = 'https://github.com/with-backed';
 export const FAQ_URL =
   'https://with-backed.notion.site/FAQ-df65a5002100406eb6c5211fb8e105cf';
+export const BUNNY_IMG_URL =
+  "https://github.com/with-backed/backed-interface/blob/main/components/Logo/backed-bunny.png?raw=true";
```

Comment:
It might be best to move the images from the [`Logo` component](https://github.com/with-backed/backed-interface/tree/main/components/Logo) into the `public/` directory and then have this URL be based on that. Would have to update the other image imports, but probably not a huge lift.

:thinking: actually we might even want to use the network-specific bunny variants in the og image, but we can leave that for future work